// rdstation/lib/RDStation/Authentication.js
"use strict";

Object.defineProperty(exports, "__esModule", {
    value: true
});

var _AccessToken = require("./AccessToken");

var _AccessToken2 = _interopRequireDefault(_AccessToken);

var _Request = require("./Request");

var _Request2 = _interopRequireDefault(_Request);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

class Authentication {
    constructor(clientId, clientSecret) {
        this.clientId = clientId;
        this.clientSecret = clientSecret;
    }

    getAccessToken(code) {
        const response = _Request2.default.send('POST', '/auth/token', {
            client_id: this.clientId,
            client_secret: this.clientSecret,
            code: code
        });

        if (!response) {
            throw new Error('Authentication::getAccessToken(): Request failed on response.');
        }

        return new _AccessToken2.default(this, response['access_token'], response['expires_in'], esponse['refresh_token']);
    }

    refreshAccessToken(refreshToken) {
        const fields = {
            client_id: this.clientId,
            client_secret: this.clientSecret,
            refresh_token: refreshToken
        };

        const response = _Request2.default.send('POST', '/auth/token', fields, {
            'Content-Type': 'application/json'
        });

        if (!response) {
            throw new Error('Could not refresh access token.');
        }

        return new _AccessToken2.default(this, response['access_token'], response['expires_in'], response['refresh_token']);
    }

    get clientId() {
        return this.clientId;
    }

    set clientId(val) {
        this.clientId = val;
    }

    get clientSecret() {
        return this.clientSecret;
    }

    set clientSecret(val) {
        this.clientSecret = val;
    }
}
exports.default = Authentication;
